### DISCUSSION TITLE
Computational efficiency about DP potential in lammps

### DISCUSSION THREAD

**Discussion by ghost** [deepmodeling/deepmd-kit#1735] [Q&A]
Hi there,

We've trained a usable potential with DEEPMD. However, use that model to do the md calculation takes 3 hours to run 100000 in a 700 atoms system with 16 cpu cores. That is much slower  than the normal md calculation, is there any method to make computing more efficient？

many thanks.

**Comment by wanghan-iapcm** [comment]
You may compress your model https://github.com/deepmodeling/deepmd-kit/blob/master/doc/freeze/compress.md

**Reply by ghost** [reply]
Many thanks for your reply, I know compressing model is helpful in the model training. Is it also useful for molecular dynamics calculations？

**Reply by wanghan-iapcm** [reply]
Compressed model does not help your training unless you already have a good model to initialize, and use the "init-frz-model" mode.

**Comment by njzjz** [comment]
Have you used MPI and limited the number of threads for each process to 1?

**Reply by ghost** [reply]
LAMMPS (29 Sep 2021 - Update 2)
OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:98)
  using 1 OpenMP thread(s) per MPI task
 4 by 4 by 1 MPI processor grid
112.3% CPU use with 16 MPI tasks x 1 OpenMP threads
I am not sure, and those are the information about the mpi and tread in my log file.

**Reply by njzjz** [reply]
`OMP_NUM_THREADS`, `TF_INTRA_OP_PARALLELISM_THREADS`, and `TF_INTER_OP_PARALLELISM_THREADS` should be set to 1. Then you can test the performance again.
